client/src/components/arabicListWords/index.js

```diff
@@ -19,10 +20,12 @@ class arabicListWords extends Component {
         <h1> List Of Arabic Words </h1>
         <React.Fragment>
           {this.state.arabicList.map(({ english, arabic, pronunciation }) => (
-            <ul>
-              <li className="list">
-                {english}
-                {arabic}
+            <ul className="list">
+              <li className="english">{english}</li>
```

Comment:
I would add word to this class name e.g arabic-word

Reply:
Ok